### PR TITLE
Fixed layout issue while adding linked products #3802

### PR DESCRIPTION
**Pull request by devansh-webkul**
**BUGS:**
Issue: https://github.com/bagisto/bagisto/issues/3802

### CODE REVIEW

**Comment by devansh-webkul** [comment]
I would recommend instead of the full name we should use short names and add  '...'.

**Comment by bhanu-webkul** [comment]
Done

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/product-links.blade.php [hunk 1]
```diff
@@ -50,8 +50,8 @@
 
             <input type="hidden" name="related_products[]" v-for='(product, index) in addedProducts.related_products' v-if="(key == 'related_products') && addedProducts.related_products.length" :value="product.id"/>
 
-            <span class="filter-tag" style="text-transform: capitalize; margin-top: 10px; margin-right: 0px; justify-content: flex-start" v-if="addedProducts[key].length">
-                <span class="wrapper" style="margin-left: 0px; margin-right: 10px;" v-for='(product, index) in addedProducts[key]'>
+            <span class="filter-tag" style="text-transform: capitalize; margin-top: 10px; margin-right: 0px; justify-content: flex-start; height: 100%;" v-if="addedProducts[key].length">
```
[the remark this answers is not in the document]
Comment (devansh-webkul) [reply packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/product-links.blade.php]: Its already removed in the latest commit.

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/product-links.blade.php [hunk 2]
```diff
@@ -50,8 +50,8 @@
...
+                <span class="wrapper" style="margin-left: 0px; margin-right: 10px; height: 100%;" v-for='(product, index) in addedProducts[key]'>
```
[the remark this answers is not in the document]
Comment (devansh-webkul) [reply packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/product-links.blade.php]: Its already removed in the latest commit.

**Pull request merged**